### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_Screenreader : Narrator doesn't announce the "Expand/Collapse" status for dropdown in "Left navigation" links.

### ISSUE THREAD

**Issue by jadhavprathmesh01** [microsoft/vscode-docs#5203]
[](https://user-images.githubusercontent.com/87858881/149924846-1118faaf-0d61-41ed-aca8-64c42857345a.gif)[](https://user-images.githubusercontent.com/87858881/149928610-a5bf96a6-8c9d-44f8-8960-b350ed64637a.png)GitHubTags:

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Narrator; #NVDA; #WCAG1.3.1; #FTP; #STP; #ChromiumEdge; #Chrome;

Environment Details:
Application Name: Visual Studio Code Services
Windows Version: Windows 11

Repro Steps:

1. Launch the application Visual Studio Code Services
2. Tab navigate to "Docs" link.
3. Left and right navigation will appear.
4. Turn on the narrator.
5. Tab navigate to the Left navigation links and observe that the narrator is announcing or no "Expand/Collapse" status for dropdown.

Actual:
The narrator is not announcing the "Expand/Collapse" status for dropdown when we navigate using keyboard.
NOTE: This issue is not reproducing in NVDA screenreader.

Expected Result:
The narrator should announce the "Expand/Collapse" status for dropdown.

Narrator attachment:

![149924846-1118faaf-0d61-41ed-aca8-64c42857345a](https://user-images.githubusercontent.com/87858881/155506476-3a92bf63-f742-47c0-aa1d-ee1df382e4b2.gif)

NVDA Screenreader attachment:
![149928610-a5bf96a6-8c9d-44f8-8960-b350ed64637a](https://user-images.githubusercontent.com/87858881/155506648-41aeeceb-0deb-4e64-9241-25ce6e9b420c.png)

A11y_Visual Studio Code Services_Docs_Left Right Navigation_Screenreader

**Comment by jadhavprathmesh01** [comment]
This is the link https://visualstudio.microsoft.com/ is an example where we can find Narrator announce collapsed status. Please check the **Download Visual Studio** dropdown where the narrator is announcing collapsed status.

**Comment by rzhao271** [comment]
Ok yep we're missing some aria attributes here.

**Comment by gregvanl** [comment]
Fixed

**Comment by jadhavprathmesh01** [comment]
I have verified the bug and issue has been solved.
![Closed](https://user-images.githubusercontent.com/87858881/157170478-3efc8bdd-d9e9-4ed3-bebe-9a6825fa5fa4.png)

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.75
![image](https://user-images.githubusercontent.com/85863999/217260929-49498f6b-7321-48a9-8a43-4b58f57a8714.png)
